WEEK 2/main.py

```diff
@@ -1,13 +1,21 @@
-# Auto detect text files and perform LF normalization
-* text=auto
 
+# My first exercise 
+'''
+(BAD IDENTATION, DID YOU RUN THIS CODE BEFORE SUBMITTING?)
+CORRECTION
+drink_number = [1, 3, 7, 9, 11]
+for drink in drink_number:
+     print (drink)
+ '''
```

Comment:
The indentation is bad, Good identation is crucial for your code to run or do what exactly you want it ti do.